### PR TITLE
Update Ruby to 2.7.2

### PR DESCRIPTION
**Pull request by GrahamJerome**
- [ ] 🐛 Bug fix
- [ ] 🎁 New feature
- [ ] 💎 Refactor
- [x] 🧹 Chore

Related: https://github.com/Zetatango/zetatango/issues/12411

Newer version of Ruby has been released with CVE fixes.

Update the repo.

- [ ] :godmode: None
- [x] :suspect: Low
- [ ] :rage3: High

- [ ] I have **added tests** to cover my changes.
- [ ] I have **adjusted e2e** to cover my changes.

@bcarr092 @dragoszt 

- [x] No screenshots are required

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.7.2
```
Comment (dragoszt) [inline .ruby-version:1]: what is this trickery?

**Review by dragoszt — APPROVED** [review]

**Review by dragoszt — COMMENTED** [review]
you need to update lib version (see previous upgrade https://github.com/Zetatango/porky_lib/pull/182)

File: .ruby-version (hunk 1 above)
Comment (GrahamJerome) [reply .ruby-version:1]: It's so `chruby` will auto switch when you navigate to the project in terminal.

**Comment by GrahamJerome** [comment]
> you need to update lib version (see previous upgrade #182)

Done, thanks!

**Review by bcarr092 — APPROVED** [review]

**Pull request merged**